lib/fs.js

```diff
@@ -856,7 +859,11 @@ function ftruncate(fd, len = 0, callback) {
 function ftruncateSync(fd, len = 0) {
   validateInt32(fd, 'fd', 0);
   validateInteger(len, 'len');
-  len = MathMax(0, len);
+
+  if (len < 0) {
+    throw new ERR_INVALID_ARG_VALUE('len', len);
+  }
+
```

Comment:
Shouldn't we throw a `ERR_OUT_OF_RANGE` instead?

```suggestion
validateInteger(len, 'len', 0);
```